### ISSUE TITLE
a clean validation set

### ISSUE THREAD

**Issue by cxxPulsar** [hbzju/PiCO#6]
Hi，
The paper mentions that “we split a clean validation set (10% of training data) from the training set to select the hyperparameters. After that, we transform the validation set back to its original PLL form and incorporate it into the training set to accomplish the final model training.” Could you tell me where that is in the code? Or in which file? Thanks for your help!
